recipes/libxmlpp/2.x.x/test_package/CMakeLists.txt

```diff
@@ -1,10 +1,7 @@
 cmake_minimum_required(VERSION 3.8)
-project(test_package)
+project(test_package LANGUAGES CXX)
 
-include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup(TARGETS)
-
-find_package(libxml++ REQUIRED CONFIG)
+find_package(libxmlpp REQUIRED CONFIG)
```

Comment:
I'm fine renaming it because the upstream only offers pc file, so we follow the project name for cmake and avoids path errors. 👍